listenbrainz/webserver/views/api.py

```diff
@@ -22,6 +22,10 @@ def submit_listen():
     Submit listens to the server. A user token (found on https://listenbrainz.org/user/import ) must
     be provided in the Authorization header!
 
+    Listens should be submitted for tracks when the user has listened to half the track or 4 minutes of
+    the track. If the user hasn't listened to 4 minutes or half the track, it doesn't fully count as a listen
```

Comment:
when the user has listened to half the track or 4 minutes of the track, **whichever is lower**
(or "whichever comes first"?)